sentry/src/test/java/io/sentry/SentryExceptionFactoryTest.kt

```diff
@@ -70,11 +71,22 @@ class SentryExceptionFactoryTest {
 
         val error = Exception("Exception")
 
-        val throwable = ExceptionMechanismException(mechanism, error, null)
+        val throwable = ExceptionMechanismException(mechanism, error, Thread.currentThread())
 
         val sentryExceptions = fixture.getSut().getSentryExceptions(throwable)
         assertEquals("anr", sentryExceptions[0].mechanism.type)
-        assertEquals(false, sentryExceptions[0].mechanism.isHandled)
+        assertFalse(sentryExceptions[0].mechanism.isHandled)
+        assertNull(sentryExceptions[0].stacktrace?.snapshot)
```

Comment:
`l`: It would be great to test this also with multiple exceptions and not just one.

Reply:
this class already has tests for an exception with multiple causes (aka inner exception)